keyboards/lovelive9/keymaps/default/keymap.c

```diff
@@ -0,0 +1,370 @@
+#include QMK_KEYBOARD_H
+#include "process_unicode.h"
+
+//#include "v1.h"
+
+//Following line allows macro to read current RGB settings
+
+extern rgblight_config_t rgblight_config;
+rgblight_config_t RGB_current_config;
+
+#define KC_KANJI KC_GRV
+
+
+enum Layer
+{
+  _QWERTY,
+  _DIA,
+  _YOU,
+  _TIKA,
+  _ZURA,
+  _RUBY,
+  _YOHANE,
+  _RIKO,
+  _MARI,
+  _KANAN,
+  _RUBY_SUB1,
+  _RUBY_SUB2,
+  _RUBY_SUB3,
+  _SCHOOL_IDOL_FESTIVAL
+};
+
+#define SEND_DIA 100
+#define SEND_YOU 101
+#define SEND_TIKA 102
+#define SEND_ZURA 103
+#define SEND_RUBY 104
+#define SEND_YOHANE 105
+#define SEND_RIKO 106
+#define SEND_MARI 107
+#define SEND_KANAN 108
+
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  DIA,
+  YOU,
+  TIKA,
+  ZURA,
+  RUBY,
+  RUBY_SUB1,
+  RUBY_SUB2,
+  RUBY_SUB3,
+  YOHANE,
+  RIKO,
+  MARI,
+  KANAN,
+  TO_SCHOOL_IDOL_FESTIVAL,
+  RETURN_SCHOOL_IDOL_FESTIVAL
+};
+
+#define _______ KC_TRNS
```

Comment:
QMK defines this keycode alias automatically. Line 61 can be deleted.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31

Reply:
I will delete Line 61, thank you.